spec/models/rushing_game_log_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe RushingGameLog do 
+
+  describe "associations" do 
+    it { should belong_to(:player) }    
+  end
+
+  describe "yards per carry" do
+    it "should calculate yards per rushing attempt rounded to 2 decimal points" do
+      rushing = create(:rushing_game_log)
+      YPC = (rushing.rushing_attempts.to_f / rushing.rushing_yards.to_f).round(2)
+
+      expect(rushing.yards_per_carry).to eq(YPC)
+    end
```

Comment:
Great to see the `yards_per_carry` method being tested in a model spec.

Consider using hardcoded values here rather than depending on a calculation and random numbers. There's a bit too much uncertainty in this test to be sure its always testing what it should be.

```diff
-    rushing = create(:rushing_game_log)
-    YPC = (rushing.rushing_attempts.to_f / rushing.rushing_yards.to_f).round(2)
-
-    expect(rushing.yards_per_carry).to eq(YPC)
+    rushing = RushingGameLog.new(rushing_attempts: 30, rushing_yards: 126)
+    expect(rushing.yards_per_carry).to eq(0.24)
```